README.md

```diff
@@ -1,5 +1,10 @@
 # 📖 <samp>EON</samp>
 
+<div align = "center">
+
+[![🇧🇷](https://raw.githubusercontent.com/stevenrskelton/flag-icon/master/png/16/country-4x3/br.png "Português")](https://github.com/saulojoab/eon/blob/main/docs/portuguese/README.md) || [![United States](https://raw.githubusercontent.com/stevenrskelton/flag-icon/master/png/16/country-4x3/us.png "United States")](https://github.com/saulojoab/eon/blob/main/README.md)
```

Comment:
como um ajuste final, acho que ficaria melhor separar as bandeiras usando só um `|` e colocar ele abaixo do `This repository is the mobile Expo React Native application of the EON manga reading app.`